app/src/main/java/com/example/courses/data/SharedPreferencesLocalSouse.kt

```diff
@@ -6,23 +6,26 @@ import com.example.courses.R
 import com.example.courses.UseCase.SharedPreferencesRepository
 import com.example.courses.entity.Person
 
-class SharedPreferencesLocalSouse(var context: Context):SharedPreferencesRepository {
+class SharedPreferencesLocalSouse(var context: Context) : SharedPreferencesRepository {
 
-    lateinit var sharedPreferences:SharedPreferences
+    lateinit var sharedPreferences: SharedPreferences
     override fun save(person: Person) {
-        sharedPreferences=context.getSharedPreferences(R.string.SPref.toString(),Context.MODE_PRIVATE)
-        val editor:SharedPreferences.Editor=sharedPreferences.edit()
-        editor.putString(R.string.SPref_Name.toString(),person.name)
-        editor.putInt(R.string.SPref_Rating.toString(),person.rating)
+        sharedPreferences =
+            context.getSharedPreferences(R.string.SPref.toString(), Context.MODE_PRIVATE)
```

Comment:
R.string.SPref - is an integer value. So, you file name and keys (SPref_Name, SPref_Rating) going go be just an integer value converted to string. It's not a good way to have such a key. Also, R.string.SPref_Name may have different values for different languages. So, it could break logic if you switch system logic. Briefly: you should have real constants here: maybe in this file, maybe, move all text constants to a separate file. 

And combination of snake case and camel case (SPref_Name) - bad practice as well.